apps/dashboard/app/javascript/dynamic_forms.js

```diff
@@ -293,9 +293,21 @@ function setValue(event, changeId) {
   const changeVal = table.get(chosenVal, undefined);
 
   if(changeVal !== undefined) {
-    const innerElement = $(`#${changeId}`);
-    innerElement.attr('value', changeVal);
-    innerElement.val(changeVal);
+    const element = document.getElementById(changeId);
+    if(element['type'] == 'checkbox') {
+      setCheckboxValue(element, changeVal);
+    } else {
+      element.value = changeVal;
+    }
+  }
+}
+
+function setCheckboxValue(checkbox, value) {
+  const affirmitiveValue = checkbox.value;
```

Comment:
Just a little thing - affirmative is spelled incorrectly here. Has no bearing on the code, though.